tests/Unit/PointwiseFunctions/EquationsOfState/TestHelpers.hpp

```diff
@@ -178,6 +174,23 @@ void check(EosType in_eos, const std::string& python_function_prefix,
                      std::make_index_sequence<EosType::thermodynamic_dim - 1>{},
                      member_args...);
 }
+template <class Eos, class DataType>
```

Comment:
Just make this part of the general test so we don't have to add it everywhere explicitly. This implementation is not scalable.